hack/test/manifests/gcp-cluster.yaml

```diff
@@ -23,6 +23,7 @@ spec:
         type: gce
       controlplane:
         count: 3
+      kubernetesVersion: "v1.16.1"
```

Comment:
I think this needs to drop the `v`

Reply:
Is this the version that get's passed to the Talos config?

Reply:
yes. so I suspect you're right. no `v`.